### ISSUE TITLE
Support non-top level containers in CRUD/other service

### ISSUE THREAD

**Issue by abhikeshav** [CiscoDevNet/ydk-gen#603]
For ydk >0.6.0, when using CRUDService and other services, using a child container or list, which is not the top-level node in a yang model is no longer supported. This functionality should be brought back.
For example:
```python
from ydk.models.cisco_ios_xe import Cisco_IOS_XE_bgp_oper as bgp_oper
from ydk.services import CRUDService
from ydk.providers import NetconfServiceProvider
 
... # connect to provider etc
 
neighbors = bgp_oper.BgpStateData.Neighbors()
crud.read(provider, neighbors)

```

now has to be changed to:

```python
from ydk.models.cisco_ios_xe import Cisco_IOS_XE_bgp_oper as bgp_oper
from ydk.services import CRUDService
from ydk.providers import NetconfServiceProvider
 
... # connect to provider etc
 
bgp_state = bgp_oper.BgpStateData()
crud.read(provider, bgp_state)
```
 
This backward incompatibility was noted in the release notes (https://github.com/CiscoDevNet/ydk-py/releases/tag/0.6.0).

 * [ ] Display error if top-level entity is not passed
 * [ ] Support non-top level entity

**Comment by skg-net** [comment]
@abhikeshav is there a target date to bring back this feature?

**Comment by abhikeshav** [comment]
@skg-net please expect this in one of the upcoming releases of YDK (`0.7.0` or `0.7.1`) in the next couple of months

**Comment by abhikeshav** [comment]
To work around this, `YFilter` can be used. See [this](http://ydk.cisco.com/py/docs/guides/crud_guide.html#reading-a-list) for example

**Comment by ygorelik** [comment]
Resolved in YDK-0.8.1
